language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -37,6 +38,31 @@ import DA.Daml.Project.Consts
 import DA.Daml.Project.Types
 import qualified DA.Daml.Project.Types as DATypes
 
+-- Referenced from 'writePackageJson'. Lifted here for easy
+-- maintenance.
+data ConfigConsts = ConfigConsts
+  { pkgDependencies :: [(T.Text, T.Text)]
+  , pkgDevDependencies :: [(T.Text, T.Text)]
+  , pkgScripts :: [(T.Text, T.Text)]
+  }
+configConsts :: T.Text -> ConfigConsts
+configConsts sdkVersion = ConfigConsts
+  { pkgDependencies =
+      [ ("@mojotech/json-type-validation", "^3.1.0")
+      , ("@daml/types", sdkVersion)
+      ]
+  , pkgDevDependencies =
+      [ ("@typescript-eslint/eslint-plugin", "2.11.0")
+      , ("@typescript-eslint/parser", "2.11.0")
+      , ("eslint", "^6.7.2")
+      , ("typescript", "~3.7.3")
+      ]
+  , pkgScripts =
+    [ ("build", "tsc --build")
+    , ("lint", "eslint --ext .ts src/ --max-warnings 0")
```

Comment:
This is unrelated to this PR but why would our users want to lint the code generated by `daml2ts`? I would argue that it’s a bug in `daml2ts` if we produce code with lint warnings.

Reply:
we think so too and test for that. the presence of this rule is needed for those tests to succeed.